src/einsteinpy/symbolic/ricci.py

```diff
@@ -22,7 +24,10 @@ def __init__(self, arr, syms, config="ll", parent_metric=None):
             Tuple of crucial symbols dentoting time-axis, 1st, 2nd, and 3rd axis (t,x1,x2,x3)
         config : str
             Configuration of contravariant and covariant indices in tensor. 'u' for upper and 'l' for lower indices. Defaults to 'll'.
-
+        parent_metric : ~einsteinpy.symbolic.metric.MetricTensor or None
+            Corrosponding Metric for the Ricci Tensor.
+            None if it should inherit the Parent Metric of Christoffel Symbols.
+            Defaults to None.
         Raises
```

Comment:
A newline before `Raises` :)

Reply:
The code for `from_riemann`, you should have figured it out from `from_christoffels` of `RiemannCurvatureTensor`
```
    @classmethod
    def from_riemann(cls, riemann, parent_metric=None):
        """
        Get Ricci Tensor calculated from Riemann Tensor

        Parameters
        ----------
        riemann : ~einsteinpy.symbolic.riemann.RiemannCurvatureTensor
           Riemann Tensor
        parent_metric : ~einsteinpy.symbolic.metric.MetricTensor or None
            Corrosponding Metric for the Ricci Tensor.
            None if it should inherit the Parent Metric of Riemann Tensor.
            Defaults to None.

        """
        if not riemann.config == "ulll":
            riemann = riemann.change_config(newconfig="ulll", metric=parent_metric)
        if parent_metric is None:
            parent_metric = riemann.parent_metric
        return cls(
            sympy.tensorcontraction(riemann.tensor(), (0, 2)),
            riemann.syms,
            config="ll",
            parent_metric=parent_metric,
        )

```

Reply:
Also you did not define the property `parent_metric`. See `riemann.py`, you would know what I'm talking about.

Reply:
Okey, I thought that just doing the tensor contraction, the Ricci Tensor would be calculated with the corresponding config. Thanks for point my error!

Reply:
It is important that Ricci Tensor is calculated by contraction when Riemann Tensor has first contravariant and rest covariant indices